src/md5crypt_long_fmt.c

```diff
@@ -363,14 +394,14 @@ struct fmt_main fmt_smd5 = {
 		MAX_KEYS_PER_CRYPT,
 		FMT_CASE | FMT_8_BIT | FMT_OMP,
 		{ NULL },
-		{ FORMAT_TAG, FORMAT_TAG1 },
+		{ FORMAT_TAG, FORMAT_TAG1, FORMAT_TAG2 },
 		smd5_tests
```

Comment:
I'd also rename `fmt_smd5`, `smd5_tests`, and `smd5_set_key`. But I understand not wanting to put more effort into this now.